config/application.rb

```diff
@@ -39,13 +39,13 @@ class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, Rack::Cors do
       allow do
-        origins /.*\.prx\.(?:org|dev)$/
+        origins /.*\.prx\.(?:org|dev|tech)$/
         resource '/api/*', methods: [:get, :put, :post, :options], headers: :any
       end
 
       allow do
         origins '*'
-        resource '/api/*', methods: [:get]
+        resource '/api/*', methods: [:get], headers: :any
```

Comment:
I can't merge this – we actually don't know what the implications of allowing anyone to make authenticated requests against the API are. We really just want to let them make unauthenticated public requests.